Request/RequestBodyParamConverter.php

```diff
@@ -95,18 +95,26 @@ public function apply(Request $request, ParamConverter $configuration)
         $this->configureContext($context = new Context(), $arrayContext);
 
         try {
-            $object = $this->serializer->deserialize(
-                $request->getContent(),
-                $configuration->getClass(),
-                $request->getContentType(),
-                $context
-            );
-        } catch (UnsupportedFormatException $e) {
-            throw new UnsupportedMediaTypeHttpException($e->getMessage(), $e);
-        } catch (JMSSerializerException $e) {
-            throw new BadRequestHttpException($e->getMessage(), $e);
-        } catch (SymfonySerializerException $e) {
-            throw new BadRequestHttpException($e->getMessage(), $e);
+            try {
+                $object = $this->serializer->deserialize(
+                    $request->getContent(),
+                    $configuration->getClass(),
+                    $request->getContentType(),
+                    $context
+                );
+            } catch (UnsupportedFormatException $e) {
+                throw new UnsupportedMediaTypeHttpException($e->getMessage(), $e);
+            } catch (JMSSerializerException $e) {
+                throw new BadRequestHttpException($e->getMessage(), $e);
+            } catch (SymfonySerializerException $e) {
+                throw new BadRequestHttpException($e->getMessage(), $e);
+            }
+        } catch (\Exception $e) {
+            if ($configuration->isOptional()) {
```

Comment:
I don't like duplications really much but I agree that it may be the best solution here :-\